edx_rest_api_client/tests/test_client.py

```diff
@@ -308,3 +309,13 @@ def auth_callback(request):
         with freeze_time(first_call_datetime + datetime.timedelta(seconds=56)):
             response = client_session.post(self.base_url + '/endpoint', data={'test': 'ok'})
             self.assertEqual(client_session.auth.token, 'cred2')
+
+    @mock.patch('edx_rest_api_client.client.requests.post')
+    def test_access_token_request_timeout_wiring2(self, mock_access_token_post):
```

Comment:
Did you want this called `...wiring2`, or is that because you were trying different things out?